test.js

```diff
@@ -15,7 +15,8 @@ describe('styled-jsx-plugin-postcss', () => {
 
   it('works with placeholders', () => {
     assert.equal(
-      plugin('p { color: %%styled-jsx-placeholder-0%%; & img { display: block; } } %%styled-jsx-placeholder-1%%'), 'p { color: %%styled-jsx-placeholder-0%% } p img { display: block; } %%styled-jsx-placeholder-1%%'
+      plugin('p { color: %%styled-jsx-placeholder-0%%; & img { display: block; } } %%styled-jsx-placeholder-1%%')
```

Comment:
we should add a test where the css has quotes and other characters that could be problematic when stringifying to JSON